setup.py

```diff
@@ -64,7 +64,7 @@
         "tensorflow~=2.6.1;python_version=='3.6'",
         "tensorflow~=2.7.0;python_version>='3.7'",
         "tf-slim>=1.1.0",
-        "torch>=1.8",  # for CUDA 11 support
```

Comment:
I think we don't need to upgrade torch because of CUDA 11
https://pytorch.org/get-started/previous-versions/#v180

Reply:
the change comes from this security issue: https://github.com/advisories/GHSA-47fc-vmwq-366v

this is also linked to the new security tests https://github.com/recommenders-team/recommenders/blob/0d17767f441f8563d9dcfdca951ceaa243b1df27/tests/security/test_dependency_security.py